### PR TITLE
add example for environment

### PR DESCRIPTION
**Pull request by stefangweichinger**

### CODE REVIEW

File: docs/ansible/roles/cron/defaults-detailed.rst [hunk 1]
```diff
@@ -361,3 +361,16 @@ Execute a custom script every week, as the ``backup`` user:
...
+.. code-block:: yaml
+
+   cron__jobs:
+
+     'script-without-mail':
+       special_time: 'hourly'
+       job: '/usr/local/lib/hourly-job'
+       enviroment:
+         - MAILTO: ''
+
```
Comment (drybjed) [inline docs/ansible/roles/cron/defaults-detailed.rst]: Remove empty line at the end of the file.

Comment (stefangweichinger) [reply docs/ansible/roles/cron/defaults-detailed.rst]: and I have a typo: "n" missing in "environment". sorry.

Comment (stefangweichinger) [reply docs/ansible/roles/cron/defaults-detailed.rst]: done, I think ...

File: docs/ansible/roles/cron/defaults-detailed.rst [hunk 2]
```diff
@@ -361,3 +361,15 @@ Execute a custom script every week, as the ``backup`` user:
...
+.. code-block:: yaml
+
+   cron__jobs:
+
+     'script-without-mail':
+       special_time: 'hourly'
+       job: '/usr/local/lib/hourly-job'
+       environment:
+         - MAILTO: ''
```
Comment (drybjed) [inline docs/ansible/roles/cron/defaults-detailed.rst]: Ah, another problem - the `environment` parameter needs to be a dictionary, not a list.

**Review by tasosalvas — DISMISSED** [review]
I like examples :)

File: docs/ansible/roles/cron/defaults-detailed.rst (hunk 2 above)
Comment (stefangweichinger) [reply docs/ansible/roles/cron/defaults-detailed.rst]: You mean: remove dash?

```
environment:
  MAILTO: ''
```

?

Comment (drybjed) [reply docs/ansible/roles/cron/defaults-detailed.rst]: Yes, exactly. You could include additional variables to make it easier to notice.

**Review by drybjed — APPROVED** [review]
Nicely done, thanks!

**Pull request merged**